hp/elitebook/845/g9/default.nix

```
@@ -0,0 +1,25 @@
+{ config, pkgs, lib, ... }:
+
+{
+  imports =
+    [
+      ../../../../common/cpu/amd
+      ../../../../common/cpu/amd/pstate.nix
+      ../../../../common/gpu/amd
+      ../../../../common/pc/laptop
+      ../../../../common/pc/laptop/acpi_call.nix
+      ../../../../common/pc/laptop/ssd
+    ];
+
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+  boot.kernelModules = [ "synaptics_usb" ];
+  boot.kernelPackages = pkgs.linuxPackagesFor pkgs.linux_latest;
```

Please replace this with a reasonable lower bound for the kernel version:

```suggestion
   boot.kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "6.3") (lib.mkDefault pkgs.linuxPackages_latest);
```

---

Thanks for the comment @Mic92 , incorporated. :smile: